pkg/storage/stores/shipper/downloads/table_manager.go

```diff
@@ -0,0 +1,172 @@
+package downloads
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/cortexproject/cortex/pkg/chunk"
+	chunk_util "github.com/cortexproject/cortex/pkg/chunk/util"
+	pkg_util "github.com/cortexproject/cortex/pkg/util"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const cacheCleanupInterval = 24 * time.Hour
```

Comment:
24h that's seems relatively big. Specially that you're checking the time last used with TTL config, worst case it's a noop. Let's pick something lower.

Reply:
How about 12h?

Reply:
yeah one hour is probably better.

Reply:
I have pushed the code with interval set to 1h already. Resolving this.